### PR TITLE
Added README.md for RFCs

### PR DESCRIPTION
**Pull request by parmus**
Link for document: https://github.com/spotify/heroic/blob/parmus/rfcs/docs/rfcs/README.md

### CODE REVIEW

File: docs/rfcs/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+both accepted and rejected.
+
+## Filename and directory structure
+
+All RFCs should be written as single documents in the /docs/rfcs directory. The filename should 
+`<ID>-<title>.md`, where ID is a 4 digit, zero-padded RFC ID and the title is a short lowercase, 
+dash-separated title. E.g. `0042-the-meaning-of-life.md`. The ID should not be assigned until the
+RFC gets either accepted or rejected; while in draft the ID should be 0000.
+
+## Structur and content
```
Comment (udoprog) [inline docs/rfcs/README.md]: Typo

File: docs/rfcs/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+### Document template
+Please use the following template for creating new RFCs. Remove any unused state fields.
+
+```markdown
+# <title>
+
+<Brief description of the problem, which this RFC tries to solve>
+
+* **Status** Draft
+* **PR** <Link back to PR, which works with this RFC>
```
Comment (gabrielgerhardsson) [inline docs/rfcs/README.md]: Would have been cool if this was possible to do via GitHub instead. Feels a bit cumbersome to have to submit a PR to edit the document to insert a link to another PR.

File: docs/rfcs/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## Working with RFC
+The workflow for working with RFC for Heroic is designed to use PRs and diffs; just like with the
+code. The workflow works as follows:
+
+1. Commit the initial RFC in a separate branch with the status `Draft`.
+1. Open an PR for merging the RFC into master
+1. Use the PR for discussing the RFC. Keep pushing changes to the RFC until it either gets
+   _accepted_, _rejected_ or _implemented_.
+1. Update the RFC with the new status, and make sure it links back to the PR.
```
Comment (gabrielgerhardsson) [inline docs/rfcs/README.md]: The link will at this point only exist in the suggested change of the PR. When does the link become useful? Am I missing something?

File: docs/rfcs/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+1. Rename the RFC document to have the next available
+1. Merge the PR into master the master branch.
```
Comment (gabrielgerhardsson) [inline docs/rfcs/README.md]: typo

File: docs/rfcs/README.md (hunk 3 above)
Comment (parmus) [inline docs/rfcs/README.md]: Even after the PR has been merged it contain the history of the discussion around the RFC. So linking back to the now closed PR is valuable to documenting the decision.

File: docs/rfcs/README.md (hunk 2 above)
Comment (parmus) [inline docs/rfcs/README.md]: Yep, but I don't know of a way to do this with GitHub.

File: docs/rfcs/README.md (hunk 3 above)
Comment (gabrielgerhardsson) [inline docs/rfcs/README.md]: Ah, got it 👍

**Pull request merged**